docs/README.md

```diff
@@ -58,23 +58,28 @@ helm repo update
 Install KubeAI and wait for all components to be ready (may take a minute).
 
 ```bash
-cat <<EOF > helm-values.yaml
-models:
-  catalog:
-    gemma2-2b-cpu:
-      enabled: true
-      minReplicas: 1
-    qwen2-500m-cpu:
-      enabled: true
-    nomic-embed-text-cpu:
-      enabled: true
-EOF
-
 helm upgrade --install kubeai kubeai/kubeai \
-    -f ./helm-values.yaml \
     --wait --timeout 10m
 ```
 
+Install some predefined models.
+
+```bash
```

Comment:
another alternative could be to use `kubectl apply -f http://`

I kind of like the idea that you always see the full model CRD that is getting applied.

Reply:
I like that idea too. Any customizations would likely need to be done with a following kubectl command or a multi-step download file, edit, apply... Thinking of setting `minReplicas: 1`

Reply:
Lets punt on this for now.